curator/run_cron.py

```diff
@@ -27,16 +27,23 @@
 
 filename = os.getenv('CURATOR_CONF_LOCATION', '/etc/curator') + '/settings'
 
-decoded = []
+decoded = {}
 with open(filename, 'r') as stream:
-    decoded = yaml.load(stream) or []
+    decoded = yaml.load(stream) or {}
 
 connection_info = '--host ' + os.getenv('ES_HOST') + ' --port ' + os.getenv('ES_PORT') + ' --use_ssl --certificate ' + os.getenv('ES_CA') + ' --client-cert ' + os.getenv('ES_CLIENT_CERT') + ' --client-key ' + os.getenv('ES_CLIENT_KEY')
 
+defaults = {'.defaults': {'delete': {'days': int(os.getenv('CURATOR_DEFAULT_DAYS', 30))}}}
+
+default_time_unit = decoded.get('.defaults', defaults).get('delete', defaults['.defaults']['delete']).keys()[0]
+default_value = int(decoded.get('.defaults', defaults).get('delete', defaults['.defaults']['delete'])[default_time_unit])
```

Comment:
Python isn't my first language but... shouldn't `decoded.get('.defaults', defaults)` be `decoded.get('.defaults', defaults['.defaults'])` ?

Also the repetition of `decoded.get('.defaults', defaults).get('delete', defaults['.defaults']['delete']` seems like something that's begging to get put in a variable... DRY